File: Programmers/Level 1/완주하지 못한 선수/Solution.py
def solution(participant, completion):
  part = {}
  for p in participant:
      part[p] = part.get(p, 0) + 1

  for c in completion:
      part[c] = part.get(c) - 1
      if part[c] == 0: del part[c]

  return list(part.keys())[0]

File: Programmers/Level 1/완주하지 못한 선수/Solution.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    
    for(int i=0;i<completion.size();i++)
        participant.push_back(completion[i]);
    
    sort(participant.begin(),participant.end());
    
    for(int i=0;i<participant.size();i+=2){
        if(i==participant.size()-1){
            answer = participant[i];
            break;
        }
        if(participant[i]==participant[i+1])
            continue;
        else{
            answer = participant[i];
            break;
        }
    }
        
    return answer;
}